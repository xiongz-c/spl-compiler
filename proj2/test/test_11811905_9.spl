int arr[2][2];

int test_11811806_9() {
    int v1[10][10];
    float v2 = 1.0;
    int v3 = v1[v2*arr[0][0]][0];
    int v4 = v1[9];
    return v3;
}