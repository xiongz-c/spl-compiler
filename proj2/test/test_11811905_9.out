Error type 12 at Line 6: indexing by non-integer
Error type 5 at Line 7: unmatching type on both sides of assignment
